### ISSUE TITLE
Clicking event links in Calendar has no effect.

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#498]
WordPress 6.1.1
Twenty Twenty Three
My Calendar plugin is the only active plugin.
Brave browser. Also tested with Chrome.

**Describe the bug**

Clicking event links in the frontend calendar has no effect.

I am seeing this frontend calendar:
![Screenshot 2022-12-18 at 11 05 46](https://user-images.githubusercontent.com/5323259/208307977-bb3882d4-00cd-4444-a9d9-ae5dcf6ff627.jpg)

Backend:
Settings -> Display.
Calendar Links: Open links as a popup.

![Screenshot 2022-12-18 at 11 07 04](https://user-images.githubusercontent.com/5323259/208308034-7bb5a2df-4b25-46af-a505-1bb04016d11d.jpg)

I might be doing something wrong, and have checked an option, but based on having the links open as a popup I am not seeing any popup.

**Comment by joedolson** [comment]
I've been revamping a number of things in there; might be worth making sure that you're in an uncached environment. Have you tested incognito?

**Comment by paaljoachim** [comment]
I added a new event in the recently downloaded Github version. (I removed the version before it and had it remove everything to do with My Calendar to get a best possible fresh start in this version.)

I added a new event. Having 2 additional copies. Clicked to publish and noticed this message:

![Screenshot 2022-12-18 at 20 10 43](https://user-images.githubusercontent.com/5323259/208329861-3c7dca28-0196-4043-a299-3c4a32de156d.jpg)

"Event added. It will now show on the calendar. No link was generated for this event. There may be an unknown error."

Testing the calendar. The various calendar event links brought up a popup modul with the single event information. These seemed to work. Other things are going on which I will bring up in a new issue. I will close this issue for now.
